handlers/bounty.go

```diff
@@ -311,7 +311,14 @@ func MakeBountyPayment(w http.ResponseWriter, r *http.Request) {
 	}
 
 	url := fmt.Sprintf("%s/payment", config.RelayUrl)
-	bodyData := fmt.Sprintf(`{"amount": %d, "destination_key": "%s"}`, amount, request.ReceiverPubKey)
+
+	var bodyData string
+	if request.RouteHint != "" {
+		bodyData = fmt.Sprintf(`{"amount": %d, "destination_key": "%s", "route_hint": "%s"}`, amount, request.ReceiverPubKey, request.RouteHint)
+	} else {
+		bodyData = fmt.Sprintf(`{"amount": %d, "destination_key": "%s"}`, amount, request.ReceiverPubKey)
+	}
```

Comment:
Looks like were doing this in two locations it might make more sense to make this a util than repeating code